src/components/combo_box/combo_box_options_list/combo_box_options_list.tsx

```diff
@@ -68,6 +68,11 @@ export type EuiComboBoxOptionsListProps<T> = CommonProps &
     'data-test-subj': string;
     activeOptionIndex?: number;
     areAllOptionsSelected?: boolean;
+    /**
+     * Creates a custom text option. You can use `{ searchValue }` inside your string to better customize your text.
```

Comment:
```suggestion
     * Creates a custom text option. You can use `{searchValue}` inside your string to better customize your text.
```

The replace doesn't allow extra spaces